### ISSUE TITLE
Map shows red colored border and nothing else

### ISSUE THREAD

**Issue by the-yadu** [airbnb/react-native-maps#1310]
Any idea as in how to get it working on IOS
I have already done these following steps.

1. Installation in fresh project with only maps and nothing else as shown in the image below
2. Ran (npm i react-native-maps@0.13.0 --save && react-native link && react-native run-ios)
3. Did manual installation as updated in installation docs.
4. Got the key from google map console and inserted in androidManifest.xml file too.
5. ran (react-native upgrade) too
6. tried with rnpm too.
7. My node version is up to date(v7.10.0) and react-native (react-native: 0.44.0)
8. Tried to work with both inline and Stylesheet provider method.
9. Have tried these steps too
![image](https://cloud.githubusercontent.com/assets/7404181/25858379/4007a092-34f9-11e7-83df-1abde0c47cd6.png)
AND
![image](https://cloud.githubusercontent.com/assets/7404181/25858401/48f896ca-34f9-11e7-8a2a-44ce89a7cb95.png)
10. MACOS(10.12.4), IOS(Simulator with v10.3 iphone6)

> Save ME from this white screen of DEATH

[
<img width="722" alt="screen shot 2017-05-09 at 6 36 39 pm" src="https://cloud.githubusercontent.com/assets/7404181/25857918/100ab4e8-34f8-11e7-9035-b0e596ad9fbc.png">
](url)

**Comment by henrikra** [comment]
Why you are installing version 0.13? Since 0.15 is available :D

**Comment by the-yadu** [comment]
It was suggested in couple of blogs to go for 0.13 as .14 was having issues. I did try with both versions. Neither worked.

**Comment by henrikra** [comment]
Try 0.15 then if 0.13 and 0.14 are not working

**Comment by the-yadu** [comment]
I am trying to use map@0.15.0 with the current image. Its no luck either.

**Comment by the-yadu** [comment]
Hurray! Issue is solved.
Not really solved but I found a workaround.

I made a test project in my friend mac and saw the maps were working just fine out of the box.

So I took that test project and copied to my system and ran `rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build/ModuleCache/* && rm -rf node_modules/ && npm cache clean && npm i`. 
It deleted the cache and rebuild the project for my system and the maps were running on my machine after that. I took my source code and pasted inside it. and woala, the project and maps are running just fine.
